README.md

```diff
@@ -2,31 +2,74 @@
 
 Eraser helps Kubernetes admins remove a list of non-running images from all Kubernetes nodes in a cluster.
 
-## Design 
-* [Design Documentation](https://docs.google.com/document/d/1Rz1bkZKZSLVMjC_w8WLASPDUjfU80tjV-XWUXZ8vq3I/edit?usp=sharing) 
+> 🚨 This project is currently in early alpha stage. Do NOT use for production.
 
 ## Getting started
 
-Create an [ImageList](config/samples/eraser_v1alpha1_imagelist.yaml) and specify the images you would like to remove manually.
-
-* `kubectl apply -f config/samples/eraser_v1alpha1_imagelist.yaml --namespace=eraser-system`
-
-This should have triggered an [ImageJob](api/v1alpha1/imagejob_types.go) that will deploy [eraser](pkg/eraser/eraser.go) pods on every node to perform the removal given the list of images. 
-
-To view the result of the removal:
-* describe ImageList CR and look at status field:
-    * `kubectl describe ImageList -n eraser-system imagelist_sample`
-
-To view the result of the ImageJob eraser pods:
-* find name of ImageJob: 
-    * `kubectl get ImageJob -n eraser-system`
-* describe ImageJob CR and look at status field:
-    * `kubectl describe ImageJob -n eraser-system [name of ImageJob]`
+Create an [ImageList](./test/e2e/test-data/eraser_v1alpha1_imagelist.yaml) and specify the images you would like to remove manually.
+
+Example:
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: eraser.sh/v1alpha1
+kind: ImageList
+metadata:
+  name: imagelist
+spec:
+  images:
```

Comment:
can you add what are the different options like digest, image with image tag? is regex allowed other than `*`?

Reply:
no regex allowed right now, just `*`